tidb-lightning/tidb-lightning-faq.md

```diff
@@ -310,3 +310,38 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
 3. 确保整个集群使用的是同一最新版本的 `tzdata` (2018i 或更高版本)。
 
     如果你使用的是 CentOS 机器，你可以运行 `yum info tzdata` 命令查看 `tzdata` 的版本及是否有更新。然后运行 `yum upgrade tzdata` 命令升级 `tzdata`。
+<<<<<<< HEAD
+=======
+
+## `[Error 8025: entry too large, the max entry size is 6291456]`
+
+**原因**：TiDB Lightning 生成的单行 KV 超过了 TiDB 的限制。
+
+**解决办法**:
+
+目前无法绕过 TiDB 的限制，只能忽略这张表，确保其它表顺利导入。
+
+## switch-mode 时遇到 `rpc error: code = Unimplemented ...`
+
+**原因**：集群中有不支持 switch-mode 的节点。目前已知的组件中，4.0.0-rc.2 之前的 TiFlash [不支持 switch-mode 操作](https://github.com/pingcap/tidb-lightning/issues/273)。
+
+**解决办法**：
+
+- 如果集群中有 TiFlash 节点，可以将集群更新到 4.0.0-rc.2 或更新版本。
+- 如果不方便升级，可以临时禁用 TiFlash。
+
+## `tidb lightning encountered error: TiDB version too old, expected '>=4.0.0', found '3.0.18'`
+
+TiDB Lightning Local-backend 只支持导入到 v4.0.0 及以上版本的 TiDB 集群。如果尝试使用 Local-backend 导入到 v2.x 或 v3.x 的集群，就会报以上错误。此时可以修改配置使用 Importer-backend 或 TiDB-backend 进行导入。
+
+部分 `nightly` 版本的 TiDB 集群的版本可能类似 4.0.0-beta.2。这种版本的 TiDB Lightning 实际支持 Local-backend，如果使用 `nightly` 版本遇到该报错，可以通过设置配置  `check-requirements = false` 跳过版本检查。在设置此参数之前，请确保 TiDB Lightning 的配置支持对应的版本，否则无法保证导入成功。
+
+## `restore table test.district failed: unknown columns in header [...]`
+
+出现该错误通常是因为 CSV 格式的数据文件不包含 header（第一行也是数据），因此需要在 TiDB Lightning 的配置文件中增加如下配置项：
+
+```
+[mydumper.csv]
+header = false
+```
+>>>>>>> 6cc27790... tools: update tools faq (#5284)
```

Comment:
@glorv Could you help determine which content is applicable for v3.0?

Reply:
fixed